examples/tutorials/flow-in-pipeline/pipeline.txt

```diff
@@ -0,0 +1 @@
+examples/flows/standard/web-classification
```

Comment:
this also depend on web-classification
and web classification depend on azure_openai.yml
however a chane in azure_openai.yml doesn't affect this pipeline.

Reply:
same with requirements.txt

Reply:
good catch, and make sense; have updated to always inject `examples/connections/azure_openai.yml`